### ISSUE TITLE
How to get the sequence of pins to do it on the canvas?

### ISSUE THREAD

**Issue by emontesinop** [neumann-mlucas/StringArt.jl#1]
Hello, when executing the code it generates an image and a gif as a result, but you can't see the sequence of pins to do it on the canvas, do you have any update on this? How do you get the sequence of pins? I would also like to know if you have an option to choose the colors that the canvas will carry, so that it is not RGB but solid colors. Thanks

**Comment by neumann-mlucas** [comment]
Hi, I had a function to plot the pin and chords positions that I used just for debugging. I just added the function in a util script. 
should work with the same CLI arguments as the main function plus a `-f plot_pins` arguments
The sequence of pins is a circle center in the image center with diameter equal to 95% of the size of the image.
Currently there's no option to use solid colors, but I think it is a good addition, I gonna try to implement it some time this week.
